### PR TITLE
Update Django 1.10 for other locales too

### PR DESCRIPTION
**Pull request by supritashankar**
Hello,

- [X] Updating to Django 1.10 for all other locales too. 

cc @helenst

### CODE REVIEW

**Comment by helenst** [comment]
Looks good :) My only comment is that there are documentation links that could also be updated in the fr and tr translations but they currently refer to 1.8 rather than 1.9 (I guess they just weren't changed for 1.9)

**Comment by supritashankar** [comment]
@helenst Good catch 👍  There were a bunch of other locales too that were still on 1.8. Including - cs, es, hu, it, ko, pl, pt, ru, sk, uk, zh

Thanks!

File: tr/whats_next/README.md [hunk 1]
```diff
@@ -19,18 +19,18 @@ Evet! Öncelikle [Django Girls Tutorial: Extensions][3] isimli diğer kitabımı
...
+Sonra ise aşağıda listelenen diğer kaynakları deneyebilirsin. Hepsini öneriyoruz!
+
+- [Django'nun resmi eğitimi][4]
+- [Kodlamaya Yeni Başlayan (New Coder) eğitimi][5]
+- [Code Academy Python kursu][6]
+- [Code Academy HTML & CSS kursu][7]
+- [Django Carrots eğitimi][8]
+- [Learn Python The Hard Way kitabı][9]
+- [Getting Started With Django görünütülü dersleri][10]
+- [Two Scoops of Django: Best Practices for Django 1.10 kitabı][11]
```
Comment (helenst) [inline tr/whats_next/README.md]: This link shouldn't be updated, there is no Two Scoops 1.10 as far as I know :)

File: fr/whats_next/README.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ Ensuite, vous pouvez essayer les ressources listées ci-dessous. Elles sont tout
 - [Django Carrots tutorial][8]
 - [Learn Python The Hard Way book][9]
 - [Getting Started With Django video lessons][10]
-- [Two Scoops of Django: Best Practices for Django 1.8 book][11]
+- [Two Scoops of Django: Best Practices for Django 1.10 book][11]
```
Comment (helenst) [inline fr/whats_next/README.md]: Please change back to 1.8

File: cs/django_installation/instructions.md [hunk 3]
```diff
@@ -101,11 +101,11 @@ OK máme všechny důležité závislosti na místě a konečně můžeme nainst
 
 ## Instalace Django
 
-Máš své `virtualenv` nastartované a můžeš nainstalovat Django pomocí příkazu `pip`. V konzoli, spusť `pip install django == 1.8` (<0>Poznámka:</0> , používáme zde dvojité rovnítko: `==`).
+Máš své `virtualenv` nastartované a můžeš nainstalovat Django pomocí příkazu `pip`. V konzoli, spusť `pip install django == 1.10` (<0>Poznámka:</0> , používáme zde dvojité rovnítko: `==`).
 
 ```
-(myvenv) ~$ pip install django==1.8
-Downloading/unpacking django==1.8
+(myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline cs/django_installation/instructions.md]: It looks like this locale was never updated to Django 1.9. The 1.8 -> 1.9 changes were less straightforward than 1.9 -> 1.10 - there were code changes with accompanying explanations which would require the involvement of translators to bring them to that point. So I suggest we should not update `cs` and any others that are still on 1.8 in this work. :smile:

File: es/deploy/README.md [hunk 4]
```diff
@@ -179,26 +179,26 @@ Esto va a descargar una copia de tu código en PythonAnywhere. Compruébalo escr
...
     Running virtualenv with interpreter /usr/bin/python3.4
     [...]
     Installing setuptools, pip...done.
-    
+
     20:20 ~ $ source myvenv/bin/activate
-    
-    (myvenv)20:20 ~ $  pip install django==1.8 whitenoise
+
+    (myvenv)20:20 ~ $  pip install django==1.10 whitenoise
```
Comment (helenst) [inline es/deploy/README.md]: Again, we would need Spanish translators to make sure the 1.8 -> 1.9 differences are handled correctly.

File: hu/django_installation/instructions.md [hunk 5]
```diff
@@ -93,14 +93,14 @@ Oké, most már minden feltétel teljesült. Végre installálhatjuk a Django-t!
 
 ## A Django telepítése
 
-Most, hogy elindítottad a `virtualenv`-et, a `pip` segítségével tudod telepíteni a Django-t. Futtasd ezt a parancsot a konzolban: `pip install django==1.8` (figyelj oda, hogy dupla egyenlőségjelet használj: `==`).
+Most, hogy elindítottad a `virtualenv`-et, a `pip` segítségével tudod telepíteni a Django-t. Futtasd ezt a parancsot a konzolban: `pip install django==1.10` (figyelj oda, hogy dupla egyenlőségjelet használj: `==`).
 
-    (myvenv) ~$ pip install django==1.8
-    Downloading/unpacking django==1.8
+    (myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline hu/django_installation/instructions.md]: Needs .hu translation for 1.8 -> 1.9 changes

File: it/django_installation/instructions.md [hunk 6]
```diff
@@ -94,14 +94,14 @@ OK, abbiamo tutte le dipendenze importanti pronte. Finalmente possiamo installar
 
 ## Installare Django
 
-Ora che hai iniziato ad utilizzare il tuo `virtualenv`, puoi installare Django usando `pip`. Nella console, esegui `pip install django==1.8` (nota che usiamo un doppio simbolo di uguale: `==`).
+Ora che hai iniziato ad utilizzare il tuo `virtualenv`, puoi installare Django usando `pip`. Nella console, esegui `pip install django==1.10` (nota che usiamo un doppio simbolo di uguale: `==`).
 
-    (myvenv) ~$ pip install django==1.8
-    Downloading/unpacking django==1.8
+    (myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline it/django_installation/instructions.md]: Needs translators for the 1.8 -> 1.9 transition

File: ko/django_installation/instructions.md [hunk 7]
```diff
@@ -94,10 +94,10 @@
 
 ## 장고 설치하기
 
-이제 `virtualenv` 가 시작되었으니, `pip`를 이용해 장고를 설치할 수 있어요. 콘솔에서 `pip install django==1.8`를 실행해보세요. (조심하세요. 이퀄기호가 두 개예요: `==`).
+이제 `virtualenv` 가 시작되었으니, `pip`를 이용해 장고를 설치할 수 있어요. 콘솔에서 `pip install django==1.10`를 실행해보세요. (조심하세요. 이퀄기호가 두 개예요: `==`).
 
-    (myvenv) ~$ pip install django==1.8
-    Downloading/unpacking django==1.8
+    (myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline ko/django_installation/instructions.md]: Needs translators for the 1.8 -> 1.9 transition

File: pl/django_installation/instructions.md [hunk 8]
```diff
@@ -93,14 +93,14 @@ W trakcie pracy ze środowiskiem wirtualnym `python` będzie automatycznie odnos
 
 ## Instalowanie Django
 
-Teraz, gdy Twój `virtualenv` jest już uruchomiony, możesz zainstalować Django za pomocą narzędzia `pip`. W konsoli uruchom polecenie `pip install django==1.8` (zwróć uwagę, że używamy podwójnego znaku równości: `==`).
+Teraz, gdy Twój `virtualenv` jest już uruchomiony, możesz zainstalować Django za pomocą narzędzia `pip`. W konsoli uruchom polecenie `pip install django==1.10` (zwróć uwagę, że używamy podwójnego znaku równości: `==`).
 
-    (myvenv) ~$ pip install django==1.8
-    Downloading/unpacking django==1.8
+    (myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline pl/django_installation/instructions.md]: Needs translators for the 1.8 -> 1.9 transition

File: pt/django_installation/README.md [hunk 9]
```diff
@@ -93,14 +93,14 @@ Ok, nós temos todas as dependências importantes no lugar. Finalmente podemos i
 
 ## Instalando o Django
 
-Agora que você tem a sua `virtualenv` iniciado, você pode instalar Django usando `pip`. No console, execute `pip install django==1.8.5` (Perceba que usamos um duplo sinal de igual: `==`).
+Agora que você tem a sua `virtualenv` iniciado, você pode instalar Django usando `pip`. No console, execute `pip install django==1.10` (Perceba que usamos um duplo sinal de igual: `==`).
 
-    (myvenv) ~$ pip install django==1.8.5
-    Downloading/unpacking django==1.8.5
+    (myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline pt/django_installation/README.md]: Needs translators for the 1.8 -> 1.9 transition

File: ru/django_installation/instructions.md [hunk 10]
```diff
@@ -93,14 +93,14 @@
 
 ## Установка Django
 
-После запуска `virtualenv` ты можешь установить Django используя `pip`. Набери в командной строке строке `pip install django==1.8` (обрати внимание на двойной знак равенства: `==`).
+После запуска `virtualenv` ты можешь установить Django используя `pip`. Набери в командной строке строке `pip install django==1.10` (обрати внимание на двойной знак равенства: `==`).
 
-    (myvenv) ~$ pip install django==1.8
-    Downloading/unpacking django==1.8
+    (myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline ru/django_installation/instructions.md]: Needs translators for the 1.8 -> 1.9 transition

File: sk/django_installation/instructions.md [hunk 11]
```diff
@@ -101,11 +101,11 @@ OK, všetko potrebné máme prichystané. Konečne môžeme nainštalovať Djang
 
 ## Inštalácia Djanga
 
-Teraz, keď máš `virtualenv` spustený, môžeš nainštalovať Django pomocou príkazu `pip`. V konzole spusti `pip install django==1.8` (všimni si, že používame dvojité rovná sa: `==`).
+Teraz, keď máš `virtualenv` spustený, môžeš nainštalovať Django pomocou príkazu `pip`. V konzole spusti `pip install django==1.10` (všimni si, že používame dvojité rovná sa: `==`).
 
 ```
-(myvenv) ~$ pip install django==1.8
-Downloading/unpacking django==1.8
+(myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline sk/django_installation/instructions.md]: Needs translators for the 1.8 -> 1.9 transition

File: zh/django_installation/instructions.md [hunk 12]
```diff
@@ -93,14 +93,14 @@
 
 ## 安装 Django
 
-既然你有了`虚拟环境` ，您可以使用 `pip`安装 Django 。在控制台中，运行 `pip install django==1.8` （注意我们使用双等于号： `==`).
+既然你有了`虚拟环境` ，您可以使用 `pip`安装 Django 。在控制台中，运行 `pip install django==1.10` （注意我们使用双等于号： `==`).
 
-    (myvenv) ~$ pip install django==1.8
-    Downloading/unpacking django==1.8
+    (myvenv) ~$ pip install django==1.10
```
Comment (helenst) [inline zh/django_installation/instructions.md]: Needs translators for the 1.8 -> 1.9 transition

**Comment by supritashankar** [comment]
@helenst I updated the djangoscoops version. Sorry my bad.
Also reverted the locales which were on 1.8

**Comment by helenst** [comment]
Hi @supritashankar - Thank you, I think this looks good now! And no need to apologise, thank you so much for taking the time to keep the tutorial up to date! :tulip: :yellow_heart: Hopefully we can encourage some translators to help get the other languages up to date :-)

**Pull request merged**